themes/digital.gov/src/scss/new/_featured-resource.scss

```diff
@@ -0,0 +1,66 @@
+@use "uswds-core" as *;
+
+.dg-featured-resource {
+  @include u-measure(6);
+
+  a,
+  a:hover {
+    text-decoration: none;
+  }
+
+  a:focus {
+    outline-offset: calc(units(2) * -1);
+  }
+
+  &__body {
+    align-items: center;
+    background-color: color("gray-5");
+    border-radius: units(3);
+    display: flex;
+    justify-content: space-between;
+    margin: units(5) 0;
+    padding: units(2) units(7);
+    transition: 0.3s;
+
+    &:hover {
+      background-color: color("gray-10");
+    }
+  }
```

Comment:
When the image is hidden on mobile, let's adjust the padding on line 22 to take up less horizontal space and maximize room for the copy. Looks better on mobile.

```
  &__body {
    align-items: center;
    background-color: color("gray-5");
    border-radius: units(3);
    display: flex;
    justify-content: space-between;
    margin: units(5) 0;
    padding: units(2) units(3);
    transition: 0.3s;

    &:hover {
      background-color: color("gray-10");
    }

    @include at-media("tablet") {
      padding: units(2) units(7);
    }
  }
```

Reply:
Great idea, thanks!